stable/dex/templates/role.yaml

```diff
@@ -8,6 +7,10 @@ metadata:
   name: {{ template "dex.fullname" . }}
   namespace: {{ .Release.Namespace }}
 rules:
+- apiGroups: ["dex.coreos.com"] # API group created by dex
```

Comment:
when `crd.create` is `true` this rule will duplicate the same rule from the ClusterRole